.cirrus.yml

```diff
@@ -298,11 +299,11 @@ ubuntu24_clang_libcpp_task:
     dockerfile: ci/ubuntu-24.04/Dockerfile
     << : *RESOURCES_TEMPLATE
   << : *CI_TEMPLATE
-  << : *SKIP_TASK_ON_PR
```

Comment:
I'd be totally fine if we run clang-tidy only after merging or just once a week, but I guess we don't care too much about the cycles?